pkgs/default.nix

```diff
@@ -76,7 +95,7 @@ in
 
       channel-finder-service = callPackage ./epnix/tools/channel-finder/service {};
 
-      inherit (final.python3Packages) lewis pyepics;
+      inherit (final.python3Packages) lewis pyepics aioca softioc;
```

Comment:
Hmm, I wonder if having a `softioc` package isn't confusing, since there's also epics-base's `softIoc` program. But then, the Python library is named `softioc`, but the GitHub project and the executable are named `pythonSoftIOC`…

What do you think? Should we keep the Python packaged named `softioc`, but the toplevel package named `pythonSoftIOC`? Or is it more confusing?

Reply:
It's gonna be inconsistent in any case, your suggestion may be the best way of splitting it